aux/config.json

```diff
@@ -1,7 +1,7 @@
 {
     "Internal": {
-        "ChargePointId": "",
-        "CentralSystemURI": "",
+        "ChargePointId": "cp001",
+        "CentralSystemURI": "wss://127.0.0.1:8443/steve/websocket/CentralSystemService/cp001",
```

Comment:
The config.json should have empty entries fpr ChargePointId and CentralSystemURI and stay more like a template in that regard